app/models/service_template_transformation_plan.rb

```diff
@@ -7,6 +7,8 @@ def request_type
     "transformation_plan"
   end
 
+  default_value_for :internal, true
```

Comment:
We would also want a `default_value_for  :internal, false` at the base class.

Reply:
do we really want such a default_value? Without default we don't need to insert a record in the reserved table.

Reply:
I would prefer this upstream once we have the real column, so it is fine to ignore while we are working on the reserves table commits.